servicex/templates/app/configmap.yaml

```diff
@@ -103,12 +103,19 @@ data:
 
     {{ if .Values.objectStore.enabled }}
     OBJECT_STORE_ENABLED = True
+    # default to using a https connection to the client unless explicitly
+    # turned off
+      {{ if .Values.objectStore.useClientTLS  }}
+    MINIO_CLIENT_SECURED = True
```

Comment:
Maybe instead of `MINIO_CLIENT_SECURED ` and `MINIO_SECURED` it would be clearer to be `MINIO_INTERNAL_SECURED` and `MINIO_EXTERNAL_SECURED` This is such a confusing topic we need all the clarity we can muster!

Reply:
The tricky part is that the differentiation here is on the client connections vs the server to minio connections.  `MINIO_SECURED` can determine whether connections within the k8s cluster is encrypted or it can determine whether connections from the k8s cluster to another site is encrypted.  So using `MINIO_INTERNAL_SECURED` and `MINIO_EXTERNAL_SECURED` might confuse admins as to how to encrypt a connection from ServiceX server components in a K8S cluster to an external minio instance living somewhere else or living outside the K8S cluster.